uiowa/uiowa
# Added icons to local tasks tabs, added weight and UIDS red color to "Delete" tab

Resolves: https://github.com/uiowa/uiowa/issues/3508

This is my first attempt at doing anything Drupal admin styling/hook-related, so go easy on me! Feedback would be super-appreciated. Willing to work through this with anyone who wants to help

<details>
Preview:

<img width="1096" alt="Screen Shot 2021-05-13 at 12 26 42 PM" src="https://user-images.githubusercontent.com/472923/118162522-7b744700-b3e6-11eb-94ba-b501b35b2f87.png">

I wasn't sure how to safely add any icons to the tabs in the "Classy" admin theme due to its tabs already having ::before pseudo elements, so the admin menu only changes the "Delete" tab to the UIDS red.

<img width="855" alt="Screen Shot 2021-05-13 at 12 26 10 PM" src="https://user-images.githubusercontent.com/472923/118162465-67c8e080-b3e6-11eb-9eab-b96b68e09a3f.png">
</details>


# How to test

`blt ds --site=sandbox.uiowa.edu`
`yarn workspace uids_base gulp --development`

After clearing your cache, you should see icons in the main editor that looks like https://github.com/uiowa/uiowa/issues/3508#issuecomment-854094772.  But like I said, once you click into "Edit" or other admin areas, the icons do go away. Would appreciate some guidance on adding those for consistency's sake.

Thanks!
